juriscraper/opinions/united_states/state/conn.py

```diff
@@ -2,47 +2,46 @@
 CourtID: conn
 Court Short Name: Conn.
 Author: Asadullah Baig <asadullahbeg@outlook.com>
-
 History:
- - 2014-07-11: created
- - 2014-08-08, mlr: updated to fix InsanityError on case_dates
- - 2014-09-18, mlr: updated XPath to fix InsanityError on docket_numbers
- - 2015-06-17, mlr: made it more lenient about date formatting
- - 2016-07-21, arderyp: fixed to handle altered site format
- - 2017-01-10, arderyp: restructured to handle new format use case that includes
+- 2014-07-11: created
+- 2014-08-08, mlr: updated to fix InsanityError on case_dates
+- 2014-09-18, mlr: updated XPath to fix InsanityError on docket_numbers
+- 2015-06-17, mlr: made it more lenient about date formatting
+- 2016-07-21, arderyp: fixed to handle altered site format
+- 2017-01-10, arderyp: restructured to handle new format use case that includes
         opinions without dates and flagged for 'future' publication
+- 2022-02-02, satsuki-chan: Fixed docket and name separator, changed super
+    class to OpinionSiteLinear
 """
 
 from datetime import date
 
 from lxml import etree
 
+from juriscraper.AbstractSite import logger
 from juriscraper.lib.string_utils import convert_date_string, normalize_dashes
-from juriscraper.OpinionSite import OpinionSite
+from juriscraper.OpinionSiteLinear import OpinionSiteLinear
 
 
-class Site(OpinionSite):
+class Site(OpinionSiteLinear):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.crawl_date = date.today()
-        self.url = "http://www.jud.ct.gov/external/supapp/archiveAROsup{year}.htm".format(
-            year=self.crawl_date.strftime("%y")
-        )
         self.court_id = self.__module__
+        current_year = date.today().strftime("%y")
```

Comment:
if this was self.year = .... you could avoid doing it in the app ct.

Reply:
Added attribute to use the same year in connappct